### PR TITLE
feat: add musdb recipe

### PR DESCRIPTION
**Pull request by Faylixe**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/musdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/sigsep/sigsep-mus-db.git
+    git_tag: "v{{ version }}"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```
Comment (marcelotrevisani) [inline recipes/musdb/meta.yaml]: I think this recipe can be ``noarch``
```suggestion
  noarch: python
```

File: recipes/musdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/musdb/meta.yaml]:
```suggestion
    - python >=3.6
```

File: recipes/musdb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/musdb/meta.yaml]:
```suggestion
    - python >=3.6
```

File: recipes/musdb/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - libsndfile
+    - ffmpeg
+    - setuptools
+    - numpy >=1.7
+    - pysoundfile >=0.9.0
+    - stempeg >=0.1.7
```
Comment (marcelotrevisani) [inline recipes/musdb/meta.yaml]: This package ``stempeg`` does not exist in ``conda-forge``. You will need to package it first

Comment (Faylixe) [reply recipes/musdb/meta.yaml]: Yes, I have seen this with CI failure. I am currently packaging it and will submit recipe quite soon. Will let you know as long as it is done ;).

Comment (Faylixe) [reply recipes/musdb/meta.yaml]: Just created PR for `stempeg` package : https://github.com/conda-forge/staged-recipes/pull/9998 and need review :)

File: recipes/musdb/meta.yaml (hunk 3 above)
Comment (Faylixe) [reply recipes/musdb/meta.yaml]: Fixed, resolving conversation.

File: recipes/musdb/meta.yaml (hunk 2 above)
Comment (Faylixe) [reply recipes/musdb/meta.yaml]: Fixed, resolving conversation.

File: recipes/musdb/meta.yaml (hunk 1 above)
Comment (Faylixe) [reply recipes/musdb/meta.yaml]: Fixed, resolving conversation.

File: recipes/musdb/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+{% set name = "musdb" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (marcelotrevisani) [inline recipes/musdb/meta.yaml]:
```suggestion
  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```

File: recipes/musdb/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    sha256: 27aac71095f919c7e0cfb526af1e6f745275645e959f759c42711354ea0fcbf7
```
Comment (marcelotrevisani) [inline recipes/musdb/meta.yaml]:
```suggestion
  sha256: 27aac71095f919c7e0cfb526af1e6f745275645e959f759c42711354ea0fcbf7
```

File: recipes/musdb/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 27aac71095f919c7e0cfb526af1e6f745275645e959f759c42711354ea0fcbf7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  # this project need libsndfile, however, there is no such package for win platform
+  skip: true  # [win]
```
Comment (chrisburr) [inline recipes/musdb/meta.yaml]: If it's noarch it's okay for windows to fail with missing dependencies:
```suggestion
  noarch: python
```

Comment (Faylixe) [reply recipes/musdb/meta.yaml]: Let's check that way, the `stempeg` requirement will sure fail for Windows as it does not have a Windows version (@see https://github.com/conda-forge/staged-recipes/pull/9998).

**Comment by Faylixe** [comment]
The CI is falling because of the missing `stempeg` dependency. As the associated PR was merged for this one, it turns out that the feedstock repository has been created but is empty. The CI process failed, and I don't know why : https://travis-ci.org/conda-forge/staged-recipes

**Comment by Faylixe** [comment]
@conda-forge/staged-recipes can you please review and merge this PR :)

**Pull request merged**